editor/src/messages/defer/defer_message_handler.rs

```diff
@@ -0,0 +1,36 @@
+use crate::messages::prelude::*;
+
+#[derive(Debug, Default, ExtractField)]
+pub struct DeferMessageHandler {
+	after_graph_run: Vec<(u64, Message)>,
+	after_viewport_resize: Vec<Message>,
+	current_graph_submission_id: u64,
```

Comment:
I don't see why we have to associate queued messages with their execution id, since they should always be run after the next execution